tasks/testChainAdapter.ts

```diff
@@ -19,8 +19,8 @@ task("testChainAdapter", "Verify a chain adapter")
   .addParam("amount", "Amount to bridge to the destination chain")
   .setAction(async function (args, hre: HardhatRuntimeEnvironment) {
     const { deployments, ethers, getChainId, network } = hre;
-    const provider = new ethers.providers.StaticJsonRpcProvider(network.config.url);
-    const signer = new ethers.Wallet.fromMnemonic(getMnemonic()).connect(provider);
+    const provider = new ethers.providers.StaticJsonRpcProvider((network.config as any).url);
```

Comment:
This suppresses the warnings but it doesn't do anything to improve safety. I'm in two minds about whether we want this.